src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -97,6 +97,28 @@
     public static final String WORKFLOWS = "workflows";
     public static final String EMAIL_FIELD = "email";
 
+    // These clients are expensive. They internally have thread pools and other resources. Let's
+    // create one instance and reuse it as much as possible. Client is thread-safe per the docs.
```

Comment:
The docs: https://www.elastic.co/guide/en/elasticsearch/client/java-rest/6.8/_changing_the_client_8217_s_initialization_code.html